packages/desktop-lib/src/lib/server/service/server-task.ts

```
@@ -102,43 +97,135 @@ export abstract class ServerTask {
 		});
 	}
 
+	/**
+	 * Handles the stdout data for the running task.
+	 *
+	 * @param {Buffer} data - Data emitted from the stdout stream.
+	 * @param {Function} resolve - Function to resolve the promise.
+	 * @param {Function} reject - Function to reject the promise.
+	 * @returns {void}
+	 */
+	private handleStdout(data: Buffer, resolve: Function, reject: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)